pyproject.toml

```diff
@@ -36,7 +36,7 @@ dependencies = [
     "incremental >= 24.7.0",
     "Automat >= 0.8.0",
     "hyperlink >= 17.1.1",
-    "attrs >= 21.3.0",
+    "attrs >= 22.2.0",
```

Comment:
When we change the dependencies here, we should also update the minimum dependency test environment in tox

https://github.com/twisted/twisted/blob/f59bb561e0393458e8c113a822f8cff8a3ed69f7/tox.ini#L80

------

I don't know why the tests were not failing on mindeps... I will investigate

Reply:
I have created a separate issue to fix this https://github.com/twisted/twisted/issues/12282